### ISSUE TITLE
[1.3.3-rc.1] Windows install-runtime.js failing, cannot read property 'async' of undefined

### ISSUE THREAD

**Issue by Victordmdb** [meteor/meteor#7181]
When I try to upgrade a current project or create a new one on Windows 10:

`meteor create --release 1.3.3-rc.1 test`

I get the following error:

```
C:\Users\v\AppData\Local\.meteor\packages\meteor-tool\1.3.3-rc.1\mt-os.windows.x86_32\tools\tool-env\install-runtime.js:18
var realAsync = regenerator.async;
                           ^
TypeError: Cannot read property 'async' of undefined
    at Object.<anonymous> (C:\Users\v\AppData\Local\.meteor\packages\meteor-tool\1.3.3-rc.1\mt-os.windows.x86_32\tools\tool-env\install-runtime.js:18:28)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (C:\Users\v\AppData\Local\.meteor\packages\meteor-tool\1.3.3-rc.1\mt-os.windows.x86_32\tools\tool-env\install-babel.js:16:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)

```

Just to be sure it wasn't to do with missing packages I ran

`npm install -g regenerator babel-runtime`

**Comment by benjamn** [comment]
Try again after `meteor update --release 1.3.3-rc.4`?

**Comment by Victordmdb** [comment]
Nope :( Create/update still get same error

**Comment by Victordmdb** [comment]
Im started getting an error now when I run any meteor app, or even create a new one. I've tested multiple versions (1.3,1.3.2,1.3.2.3,1.3.2.4), and this is in a newly created project. This started around the same time i was trying to upgrade to the 1.3.3 rc above, but I can't be sure that is the cause.

```
W20160608-13:18:25.194(8)? (STDERR) C:\OneDrive\Development\testproj\.meteor\local\build\programs\server\shell-server.js:326
W20160608-13:18:25.194(8)? (STDERR) var evalCommandPromise = Promise.resolve();
W20160608-13:18:25.194(8)? (STDERR)                                  ^
W20160608-13:18:25.194(8)? (STDERR) TypeError: Object #<Object> has no method 'resolve'
W20160608-13:18:25.194(8)? (STDERR)     at Object.<anonymous> (C:\OneDrive\Development\testproj\.meteor\local\build\programs\server\shell-server.js:326:34)
W20160608-13:18:25.194(8)? (STDERR)     at Module._compile (module.js:456:26)
W20160608-13:18:25.195(8)? (STDERR)     at Object.Module._extensions..js (module.js:474:10)
W20160608-13:18:25.195(8)? (STDERR)     at Module.load (module.js:356:32)
W20160608-13:18:25.195(8)? (STDERR)     at Function.Module._load (module.js:312:12)
W20160608-13:18:25.196(8)? (STDERR)     at Module.require (module.js:364:17)
W20160608-13:18:25.196(8)? (STDERR)     at require (module.js:380:17)
W20160608-13:18:25.196(8)? (STDERR)     at Object.<anonymous> (C:\OneDrive\Development\testproj\.meteor\local\build\programs\server\boot.js:101:3)
W20160608-13:18:25.196(8)? (STDERR)     at Module._compile (module.js:456:26)
W20160608-13:18:25.196(8)? (STDERR)     at Object.Module._extensions..js (module.js:474:10)
```

**Comment by scheung38** [comment]
After  meteor update --release 1.3.3-rc.4 my app breaks...

W20160609-14:56:51.927(1)? (STDERR) /Users/seb/.meteor/packages/meteor-tool/.1.3.3-rc.4.123jw7s++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20160609-14:56:51.928(1)? (STDERR)                                             throw(ex);
W20160609-14:56:51.928(1)? (STDERR)                                                   ^
W20160609-14:56:52.008(1)? (STDERR) ReferenceError: FutureTasks is not defined
W20160609-14:56:52.009(1)? (STDERR)     at meteorInstall.lib.collections.js (lib/collections.js:57:1)
W20160609-14:56:52.010(1)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160609-14:56:52.010(1)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160609-14:56:52.011(1)? (STDERR)     at /Users/seb/.meteor/local/build/programs/server/app/app.js:918:1
W20160609-14:56:52.011(1)? (STDERR)     at /Users/seb//.meteor/local/build/programs/server/boot.js:291:10
W20160609-14:56:52.011(1)? (STDERR)     at Array.forEach (native)
W20160609-14:56:52.015(1)? (STDERR)     at Function._.each._.forEach (/Users/seb/.meteor/packages/meteor-tool/.1.3.3-rc.4.123jw7s++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160609-14:56:52.015(1)? (STDERR)     at /Users/seb/.meteor/local/build/programs/server/boot.js:133:5
=> Exited with code: 8

**Comment by cypa1** [comment]
Same here. Had to stick with previous version found on the machine (1.3.2_4)

**Comment by benjamn** [comment]
This should be fixed in Meteor 1.3.3.1, which you can try by running `meteor update --release 1.3.3.1`.
